test/server/lib/validation/index.test.js

```diff
@@ -19,71 +18,61 @@ describe.skip('server/lib/validation/applications', () => {
     done();
   });
 
-  it('should validate a correctly formatted contact name', () => {
-    expect(validator.isValidContactName('valid name')).to.eventually.equal(true);
-    expect(validator.isValidContactName('valid hyphenated-name')).to.eventually.equal(true);
-    expect(validator.isValidContactName('valid quoted\'name')).to.eventually.equal(true);
+  it('should validate a correctly formatted app  name', () => {
+    expect(validator.isValidAppName('app name')).to.equal(true);
+    expect(validator.isValidAppName('valid hyphenated-name')).to.equal(true);
+    expect(validator.isValidAppName('valid quoted\'name')).to.equal(true);
     expect(stubLogger).to.have.been.calledThrice;
   });
-
-  it('should validate and return an error if contact name is not correctly formatted', () => {
-    const errors = {};
-    errors.fullName = errorManifest.fullName.incorrect;
-    expect(validator.isValidContactName('-日女子أَبْجَدِيّ')).to.be.rejectedWith(errors);
+  it('should validate and return an error for blank app name', () => {
+    expect(validator.isValidAppName('')).to.equal(false);
```

Comment:
Any reason that you didn't include the other checks that were equivilent for the old 'isValidContactName'?

Reply:
sorry I thought Id put them in